blockchain/vm/internaltx_tracer.go

```diff
@@ -174,20 +169,29 @@ func (this *InternalTxTracer) CaptureStart(from common.Address, to common.Addres
 // tracerLog is used to help comparing codes between this and call_tracer.js
 // by following the conventions used in call_tracer.js
 type tracerLog struct {
-	env      *vm.EVM
+	env      *EVM
 	pc       uint64
-	op       vm.OpCode
+	op       OpCode
 	gas      uint64
 	cost     uint64
-	memory   *vm.Memory
-	stack    *vm.Stack
-	contract *vm.Contract
+	memory   *Memory
+	stack    *Stack
+	contract *Contract
 	depth    int
 	err      error
 }
 
+func wrapError(context string, err error) error {
+	var message string
+	switch err := err.(type) {
+	default:
+		message = err.Error()
+	}
```

Comment:
What is the purpose of this code fragment?

Reply:
Oh, I didn't take a look into the function. It is copied from `tracer.go`

Reply:
Fixed by de121a2633467bd8710b4f9b0ebf08cfef0ad903